go/enclave/rpc_server.go

```diff
@@ -266,3 +271,27 @@ func (s *server) decodeBlock(encodedBlock []byte) types.Block {
 	}
 	return block
 }
+
+// serializeEVMError deserializes EVM errors into the RPC response
```

Comment:
it serialises, not de-serialises

Reply:
My linter keeps saying it's `SerialiZation` lol. good spot 👍